lib/shopify_app/scripttags_manager.rb

```diff
@@ -57,5 +60,11 @@ def scripttag_exists?(src)
     def current_scripttags
       @current_scripttags ||= ShopifyAPI::ScriptTag.all.index_by(&:src)
     end
+
+    def build_src(src)
+      src.respond_to?(:call) ? src.call(shop_domain) : src
+    rescue => e
+      raise CreationFailed, "cannot create script src: #{e.message}"
```

Comment:
I agree with @alexcoco here let's just raise the original. If this involved network or something I could see wrapping it since a lot of failures could come through here with no actionable changes but this is likely to be syntax error stuff imo.